assignments/arrays.js

```diff
@@ -63,34 +63,52 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+let car33 = {};
+for (let i = 0; i < inventory.length; i++) {
+  if (inventory[i].id === 33) {
+    car33 = {...inventory[i]};
+  };
+}
```

Comment:
Nice job. Interesting use of the spread operator here. If you wanted to define a variable to identify the car's id that you are searching for, a simpler method would then be to use find(): 

```suggestion
let id = 33;
const car = inventory.find(car => car.id === id);
console.log(`Car ${id} is a ${car.car_year} ${car.car_make} ${car.car_model}`);
```

Reply:
Ah, good call on using `.find()` instead. Thanks!